dbms/src/Core/SpillHandler.h

```diff
@@ -62,4 +63,29 @@ class SpillHandler
     static const Int64 INVALID_CURRENT_SPILLED_FILE_INDEX = -10;
 };
 
+class CachedSpillHandler
```

Comment:
Move `CachedSpillHandler` to a seperate file?

Reply:
ok, moved.